packages/synapse-react-client/src/assets/icons/Sort.tsx

```diff
@@ -21,7 +21,7 @@ const SortIcon: React.FunctionComponent<
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
-    {(direction as string).toUpperCase() === Direction.DESC ? (
+    {(direction as string).toUpperCase() === 'DESC' ? (
```

Comment:
New lint rule checks that we only compare enum to enum and string to string. Since `direction` is a string, we should compare it to the string `'DESC'`.